src/coreclr/jit/gentree.h

```diff
@@ -571,6 +571,10 @@ enum GenTreeFlags : unsigned int
     GTF_MDARRLEN_NONFAULTING    = 0x20000000, // GT_MDARR_LENGTH -- An MD array length operation that cannot fault. Same as GT_IND_NONFAULTING.
 
     GTF_MDARRLOWERBOUND_NONFAULTING = 0x20000000, // GT_MDARR_LOWER_BOUND -- An MD array lower bound operation that cannot fault. Same as GT_IND_NONFAULTING.
+
+    GTF_VECCON_FROMSCALAR       = 0x80000000,   // GT_VECCON -- Indicate the vector constant is created from the same scalar.
+
+    GTF_BROADCAST_EMBEDDED      = 0x80000000   //  GT_HWINTRINSIC -- Indicate this broadcast node is part of embedded broadcast.
```

Comment:
Is this needed? Shouldn't the fact that the broadcast is contained be enough to indicate this?

I'm not aware of any other places where we'd be containing a broadcast intrinsic today.